### ISSUE TITLE
why Can not read tiles from a striped image?

### ISSUE THREAD

**Issue by Starzhangssss** [MWod/DeeperHistReg#18]
I used openslides.loader to load my svs file, and the saver I used was tiff.saver, but an error occurred:
raise Error('unable to call {0}'.format(operation_name))
pyvips.error.Error: unable to call tiffsave
source input: Can not read tiles from a striped image

If pil.saver is used, an error will occur:
struct.error: 'L' format requires 0 <= number <= 4294967295

完整报错（ tiff.saver）：

![Image](https://github.com/user-attachments/assets/e49d8cab-21cd-498e-bd67-69c8745e3443)

**Comment by Starzhangssss** [comment]
my Parameters.json:

{"device": "cuda:0", "echo": false, "save_final_images": true, "save_final_displacement_field": true, "case_name": "Default_Nonrigid_test1", "logging_path": null, "loading_params": {"final_level": 0, "pad_value": 255.0, "sloader": "openslide", "tloader": "openslide","source_resample_ratio": 0.2, "target_resample_ratio": 0.2}, "saving_params": {"saver": "pil", "final_saver": "tiff", "save_params": "pil", "extension": ".tiff", "final_extension": ".tiff"}, "preprocessing_params": {"preprocessing_function": "basic_preprocessing", "initial_resampling": true, "initial_resolution": 4096, "normalization": true, "convert_to_gray": true, "clahe": true, "save_results": true, "flip_intensity": true}, "run_initial_registration": true, "initial_registration_params": {"save_results": true, "initial_registration_function": "multi_feature", "registration_size": 620, "registration_sizes": [150, 200, 250, 300, 350, 400, 450, 500], "transform_type": "rigid", "keypoint_threshold": 0.005, "match_threshold": 0.3, "sinkhorn_iterations": 50, "show": false, "angle_step": 90, "cuda": true, "device": "cuda:0", "echo": false, "run_sift_ransac": true, "run_superpoint_superglue": true, "run_superpoint_ransac": false}, "run_nonrigid_registration": true, "nonrigid_registration_params": {"save_results": true, "nonrigid_registration_function": "instance_optimization_nonrigid_registration", "device": "cuda:0", "echo": false, "cost_function": "ncc_local", "cost_function_params": {"win_size": 7}, "regularization_function": "diffusion_relative", "regularization_function_params": {}, "registration_size": 4096, "num_levels": 8, "used_levels": 8, "iterations": [100, 100, 100, 100, 100, 100, 100, 200], "learning_rates": [0.005, 0.0025, 0.0025, 0.0025, 0.0025, 0.0025, 0.0025, 0.0015], "alphas": [1.5, 1.5, 1.5, 1.5, 1.5, 1.5, 1.5, 1.8]}}

**Comment by MWod** [comment]
It seems that your files are not supported by PyVips and OpenSlide. I suggest to first convert your dataset to .tiff and then attempt to register them again.
